plugin/core/sessions.py

```diff
@@ -71,11 +71,13 @@ def with_client(client: Client) -> 'Session':
     return session
 
 
-def get_initialize_params(project_path: str, config: ClientConfig) -> dict:
+def get_initialize_params(workspace_folders: 'List[WorkspaceFolder]', config: ClientConfig) -> dict:
+    first_folder = workspace_folders[0] if workspace_folders else None
     initializeParams = {
         "processId": os.getpid(),
-        "rootUri": filename_to_uri(project_path),
-        "rootPath": project_path,
+        "rootUri": first_folder.uri() if first_folder else None,
```

Comment:
Possible breaking change: Sending `None` for rootUri was always allowed according to the spec but we never did it.